crates/bevy_pbr/src/render/light.rs

```diff
@@ -600,6 +593,15 @@ pub fn prepare_lights(
 
     let mut point_lights: Vec<_> = point_lights.iter().collect::<Vec<_>>();
 
+    #[cfg(not(feature = "webgl"))]
+    let max_point_light_shadow_maps = point_lights
+        .iter()
+        .filter(|light| light.1.shadows_enabled)
+        .count()
+        .min((render_device.limits().max_texture_array_layers / 6) as usize);
+    #[cfg(feature = "webgl")]
```

Comment:
I thought this kind of `#[cfg()]` on an expression was a nightly feature?

Reply:
well... no